scripts/pylib/twister/twisterlib.py

```diff
@@ -2217,7 +2221,11 @@ def run_cmake(self, args=[]):
         if out:
             with open(os.path.join(self.build_dir, self.log), "a") as log:
                 log_msg = out.decode(sys.getdefaultencoding())
```

Comment:
I added it, but it doesn't work. Actually we did not meet this problem previous. This is in cmake build stage, and we got an exception here since we start to use logging subsystem v2. Any idea for this?